app/src/main/res/layout/fragment_blinds_settings.xml

```diff
@@ -11,4 +12,32 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content" />
 
+
+    <!-- add blinds button -->
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
```

Comment:
Android studio gives a warning about setting a content description. Please do this, or supress the warning if you think it is unnecessary. Please do not ignore warnings.